src/ducks/brandDictionary/withBrands.jsx

```diff
@@ -1,10 +1,23 @@
-import { connect } from 'react-redux'
+import { useMemo } from 'react'
+import { connect, useStore } from 'react-redux'
 
 import { getInstalledBrandsFromQuery } from './selectors'
 
-const withBrands = (options = { queryName: 'triggers' }) =>
-  connect(state => ({
+const mapStateToProps =
```

Comment:
we don't need this function anymore. Let's inline it and simplify 🙏

Reply:
si elle est utilisé dans withBrands et useBrands